src/AccordionItem/AccordionItem.js

```diff
@@ -14,8 +14,8 @@ type AccordionItemProps = ElementProps<'div'> & {
     accordionStore: AccordionContainer,
 };
 
-class AccordionItem extends Component<AccordionItemProps, *> {
-    componentDidMount() {
+class AccordionItem extends Component<AccordionItemProps> {
+    componentWillMount() {
```

Comment:
Why the change from didMount to willMount?

Reply:
Rebased away!